package.json

```diff
@@ -22,8 +22,7 @@
   ],
   "scripts": {
     "bug-report-test": "playwright test --config ./integration/playwright.config.ts integration/bug-report-test.ts",
-    "build": "rollup -c && tsc -b",
-    "postbuild": "node scripts/copy-build-to-dist.mjs",
```

Comment:
Was this change intentional?

Reply:
yes, otherwise type checking could fail due to tsc using outdated files - but i can move that to a dedicated PR